<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPatient</class>
 <widget class="QDialog" name="EditPatient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #8E9DCC
}</string>
  </property>
  <widget class="QComboBox" name="bloodTypeCB">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: #C6CCE8;
    color: #000000;
    border-radius: 0.5em;
	font-family: 'Gogh';
	font-weight: 500;
	font-size: 15px;
}

QComboBox:hover {
    background-color: #A9B9E3;
	font-family: 'Gogh';
	font-weight: 500;
}
</string>
   </property>
   <item>
    <property name="text">
     <string>Undefined</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>I</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>II</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>III</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IV</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="surnameTextEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color: #C6CCE8;
    color: #000000;
    border-radius: 0.5em;
	font-family: 'Gogh';
	font-weight: 500;
	font-size: 15px;
}

QTextEdit:hover {
    background-color: #A9B9E3;
	font-family: 'Gogh';
	font-weight: 500;
}
</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="lowerPressureTextEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color: #C6CCE8;
    color: #000000;
    border-radius: 0.5em;
	font-family: 'Gogh';
	font-weight: 500;
	font-size: 15px;
}

QTextEdit:hover {
    background-color: #A9B9E3;
	font-family: 'Gogh';
	font-weight: 500;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="ageLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
font-size: 20px;
font-family: &quot;Gogh&quot;;
font-weight: 500;</string>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
  </widget>
  <widget class="QLabel" name="rhFactorLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
font-size: 20px;
font-family: &quot;Gogh&quot;;
font-weight: 500;</string>
   </property>
   <property name="text">
    <string>Rh factor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #C6CCE8;
    color: #000000;
    border-radius: 0.5em;
	font-family: 'Gogh';
	font-weight: 500;
	font-size: 15px;
}
QPushButton:hover {
    background-color: #A9B9E3;
	font-family: 'Gogh';
	font-weight: 500;
}</string>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="upperPressureTextEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color: #C6CCE8;
    color: #000000;
    border-radius: 0.5em;
	font-family: 'Gogh';
	font-weight: 500;
	font-size: 15px;
}

QTextEdit:hover {
    background-color: #A9B9E3;
	font-family: 'Gogh';
	font-weight: 500;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="surnameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
font-size: 20px;
font-family: &quot;Gogh&quot;;
font-weight: 500;</string>
   </property>
   <property name="text">
    <string>Surname</string>
   </property>
  </widget>
  <widget class="QLabel" name="pulseLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>255</y>
     <width>101</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
font-size: 20px;
font-family: &quot;Gogh&quot;;
font-weight: 500;</string>
   </property>
   <property name="text">
    <string>Heart rate</string>
   </property>
  </widget>
  <widget class="QComboBox" name="rhFactorCB">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: #C6CCE8;
    color: #000000;
    border-radius: 0.5em;
	font-family: 'Gogh';
	font-weight: 500;
	font-size: 15px;
}

QComboBox:hover {
    background-color: #A9B9E3;
	font-family: 'Gogh';
	font-weight: 500;
}
</string>
   </property>
   <item>
    <property name="text">
     <string>Undefined</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="upperLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>181</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
font-size: 20px;
font-family: &quot;Gogh&quot;;
font-weight: 500;</string>
   </property>
   <property name="text">
    <string>Diastolic pressure</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="pulseCB">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>259</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
    background-color: #C6CCE8;
    color: #000000;
    border-radius: 0.5em;
	font-family: 'Gogh';
	font-weight: 500;
	font-size: 15px;
}

QSpinBox:hover {
    background-color: #A9B9E3;
	font-family: 'Gogh';
	font-weight: 500;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="bloodTypeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>95</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
font-size: 20px;
font-family: &quot;Gogh&quot;;
font-weight: 500;</string>
   </property>
   <property name="text">
    <string>Blood type</string>
   </property>
  </widget>
  <widget class="QLabel" name="lowerLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>171</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
font-size: 20px;
font-family: &quot;Gogh&quot;;
font-weight: 500;</string>
   </property>
   <property name="text">
    <string>Systolic pressure</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ageSpinBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
    background-color: #C6CCE8;
    color: #000000;
    border-radius: 0.5em;
	font-family: 'Gogh';
	font-weight: 500;
	font-size: 15px;
}

QSpinBox:hover {
    background-color: #A9B9E3;
	font-family: 'Gogh';
	font-weight: 500;
}
</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
